pytest_picked.py

```diff
@@ -18,7 +21,8 @@ def pytest_collection_modifyitems(items, config):
     if not picked_plugin:
         return
 
-    picked_files, picked_folders = _affected_tests()
+    test_files = config._getini('python_files')
```

Comment:
Maybe we should make it more implicit like `test_file_convention`. When I read `test_files` sounds like the tests files instead of the convention adopted.

Reply:
agreed. changed.